app/src/main/java/com/example/android2022/ui/CreationFragment.kt

```diff
@@ -0,0 +1,119 @@
+package com.example.android2022.ui
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.location.LocationManager
+import android.os.Bundle
+import android.view.View
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import androidx.core.content.ContextCompat.getSystemService
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.lifecycleScope
+import com.example.android2022.R
+import com.example.android2022.data.Note
+import com.example.android2022.data.NoteRepository
+import com.example.android2022.databinding.FragmentCreateBinding
+import kotlinx.coroutines.launch
+import java.text.SimpleDateFormat
+import java.util.*
+import java.util.jar.Manifest
+
+class CreationFragment : Fragment(R.layout.fragment_create) {
+
+    private var binding: FragmentCreateBinding? = null
+    private var repository: NoteRepository? = null
+    private var id: Int? = null
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding = FragmentCreateBinding.bind(view)
+        repository = NoteRepository(view.context)
+
+        id = arguments?.getString(CREATION_FRAGMENT_TAG)?.toInt()
+
+        //если редактировать
+        if (id != null) {
```

Comment:
arguments?.getString(CREATION_FRAGMENT_TAG)?.toInt()?.let {
 lifecycleScope.launch {
                val note = repository?.get(it)
                ...
                }
} ?: binding?.run {

}